### PR TITLE
PMM-11616

### PR DESCRIPTION
**Pull request by peterSirotnak**
https://jira.percona.com/browse/PMM-0

- [ ] Links to relevant pull requests

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3087-da7b06d
client docker - perconalab/pmm-client-fb:PR-3087-da7b06d
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3087-da7b06d.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3087-da7b06d&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3087-da7b06d.tar.gz

**Comment by peterSirotnak** [comment]
Workflow run: 
https://github.com/Percona-Lab/pmm-submodules/actions/runs/4280937287

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3087-5db7777
client docker - perconalab/pmm-client-fb:PR-3087-5db7777
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3087-5db7777.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3087-5db7777&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3087-5db7777.tar.gz

File: .github/workflows/pmm2-ui-tests-fb.yml [hunk 1]
```diff
@@ -131,3 +131,16 @@ jobs:
       sha: ${{ inputs.sha || github.event.pull_request.head.sha ||  'null' }}
       client_flags: '--addclient=ps,1 --addclient=modb,1 --addclient=pdpgsql,1'
       tags_for_tests: '@user-password'
+
+  rbac:
+    name: RBAC UI tests
+    uses: percona/pmm-ui-tests/.github/workflows/pmm-ui-tests.yml@PMM-11616-matrix-pipelines
```
Comment (yurkovychv) [inline .github/workflows/pmm2-ui-tests-fb.yml]: please update the branch

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3087-31a00d8
client docker - perconalab/pmm-client-fb:PR-3087-31a00d8
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3087-31a00d8.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3087-31a00d8&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3087-31a00d8.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3087-91844d1
client docker - perconalab/pmm-client-fb:PR-3087-91844d1
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3087-91844d1.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3087-91844d1&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3087-91844d1.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3087-0b1cc06
client docker - perconalab/pmm-client-fb:PR-3087-0b1cc06
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3087-0b1cc06.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3087-0b1cc06&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3087-0b1cc06.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3087-a1fdedb
client docker - perconalab/pmm-client-fb:PR-3087-a1fdedb
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3087-a1fdedb.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3087-a1fdedb&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3087-a1fdedb.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3087-a1fdedb
client docker - perconalab/pmm-client-fb:PR-3087-a1fdedb
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3087-a1fdedb.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3087-a1fdedb&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3087-a1fdedb.tar.gz

**Comment by peterSirotnak** [comment]
New Workflow run:
https://github.com/Percona-Lab/pmm-submodules/actions/runs/4314596595

**Review by yurkovychv — APPROVED** [review]

**Pull request merged**